articles/deploying-crowdstrike-with-fleet.md

```diff
@@ -0,0 +1,139 @@
+
+`crowdstrike-web-filter.mobileconfig` - This payload configures web filtering capabilities. It allows CrowdStrike to monitor network traffic at the socket level (FilterSockets is `true`) while not filtering individual packets (FilterPackets is `false`). A key component to comprehensive device protection, the filter is properly validated with Apple's security requirements and operates at the firewall level.
+
+`crowdstrike-full-disk-access.mobileconfig` - This privacy preference payload grants full disk access to CrowdStrike Falcon. It uses the CrowdStrike Apple Developer team identifier to grant the Falcon application the necessary macOS entitlements for modifying Privacy controls.
+
+### Installer
+
+From your Falcon console, click **Host setup and management** > **Sensor Downloads**. Click **Download** for the appropriate OS and architecture.
+
+From the **Software** tab in Fleet, select **Add software** > **Custom package**. Upload the installer from the previous step. Select **Automatic install** or **Self-service** if those options apply to your environment. 
+
+>Working with different hardware architectures? Use labels to scope installs based on hardware.
+
+### Post-Install Script
+
+The **Customer ID** is used to assign hosts to your tenant and validate the CrowdStrike Falcon license using a script that calls the `falconctl` binary. Your Customer ID can be found on the Sensor download page. In Fleet, define the following post-install script for the CrwdStrike Falcon installer:
+
+```
+#!/bin/bash
+CUSTOMER_ID="YOUR-CUSTOMER-ID-HERE"
+FALCON_PATH="/Applications/Falcon.app/Contents/Resources/falconctl"
+
+sudo "$FALCON_PATH" license "$CUSTOMER_ID"
+
+# Check status
+if [ $? -eq 0 ]; then
+    echo "Activation completed"
+else
+    echo "Activation failed"
+    exit 1
+fi
+```
+
+>If your org is using GitOps and you want to pass the site key as a secret, follow this guide: https://fleetdm.com/guides/secrets-in-scripts-and-configuration-profiles
+
+>For admins that are leveraging the macOS Setup Experience in Fleet, we recommend adding the software to the list of items installed on first boot.
+
+## Windows
+
+CrowdStrike offers `.exe` and `.msi` Falcon installers for Windows. Using the `.msi` is preferred as this installer type performs a silent, fully-automated installation when using the **Automatic install** option in Fleet.
+
+### Installer + script
+
+After downloading the latest CrowdStrike Falcon `.msi` installer and retrieving your Customer ID from your admin console, navigate to the **Software** tab in Fleet, then select **Add software** > **Custom package**. Upload the installer from the previous step. Select **Automatic install** or **Self-service** if those options apply to your environment. 
```

Comment:
```suggestion
1. In the Falcon console, click **Host setup and management** > **Sensor Downloads**. 
2. Download the installer for the appropriate OS and architecture.
3. In Fleet, go to **Software > Add software > Custom package** to upload the installer
4. Select **Automatic install** or **Self-service** if those options apply to your environment.

> Use [labels](https://fleetdm.com/guides/managing-labels-in-fleet) to scope installs for different hardware architectures.
```

Reply:
Changed prose style instructions to step-by-step. I also assumed that this step is the same for Windows and macOS users.